lib/utils.go

```diff
@@ -0,0 +1,25 @@
+package lib
+
+import (
+	"fmt"
+	"os"
+)
+
+// Check will throw a panic if the err is not nil
+func Check(e error) {
```

Comment:
I'm not 100% sure about the use of this check function. I guess one advantage is to have a single place to update the verification of errors?

Also I have seen a lot of this:
```go
err := foo()
lib.Check(err)
```
Where you could do instead:
```go
lib.Check(foo())
```